docs/reference/ml/trained-models/apis/put-trained-models.asciidoc

```diff
@@ -701,6 +701,32 @@ the platform identifiers used by Elasticsearch, so one of, `linux-x86_64`,
 For portable models (those that work independent of processor architecture or
 OS features), leave this field unset.
 
+//Begin prefix_strings
+`prefix_strings`::
+(Optional, object)
+Certain NLP models are trained in such a way that a prefix string should
+be applied to the input text before the input is evaluated. The prefix
+may be different depending on the intention. For asymmetric tasks such
+as infromation retrieval the prefix applied to a passage as it is indexed
+can be different to the prefix applied when searching those passages.
+
+`prefix_strings` has 2 options, a prefix string that is always applied
+in the search context and one that is always applied when ingesting the
+docs. Both are optional.
++
+.Properties of `prefix_strings`
+[%collapsible%open]
+====
+`search`:::
+(Optional, string)
+The prefix string to prepend to the input text for requests
+originating from a search query.
+`ingest`:::
+(Optional, string)
+The prefix string to prepend to the input text for requests
+at ingest where the Inference ingest processor is used. // TODO is there a shortcut for Inference ingest processor?
```

Comment:
```suggestion
at ingest where the {infer} ingest processor is used. // TODO is there a shortcut for Inference ingest processor?
```

Reply:
There is no attribute to shorten `inference ingest processor`. Do you want me to create one? Or `{infer}` for inference is enough?